client/src/components/page/Loading/index.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
```

Comment:
Do we need this component, `client/src/components/page/Loading/index.tsx` ? Since we are now using `@opensrp/loader` we should remove this

Reply:
@kelvin-muchiri Yes we are directly using @opensrp/loader, but I've created this component as a wrapper which will work with the same props across all the UI and packages. Just to keep things in a place.